[Doc/Serve] Add Arize integration (#30907)
ray-project/ray: doc/source/serve/production-guide/monitoring.md

```

See the [Ray Metrics documentation](ray-metrics) for more details, including instructions for scraping these metrics using Prometheus.

## Exporting metrics into Arize

> Besides using Prometheus to check out Ray metrics, Ray Serve also has the flexibility to export the metrics into other observability platform.

Reviewer: ```suggestion
Besides using Prometheus to check out Ray metrics, Ray Serve also has the flexibility to export the metrics into other observability platforms.
```

> [Arize](https://docs.arize.com/arize/) is the machine learning observability platform which can help you to monitor real-time model performance, root cause model failures/performance degradation using explainability & slice analysis and surface drift, data quality, data consistency issues etc.

Reviewer 2: ```suggestion
[Arize](https://docs.arize.com/arize/) is the machine learning observability platform which can help you to monitor real-time model performance, root cause model failures or performance degradation using explainability & slice analysis and surface drift, data quality, data consistency issues, and much more.
```

> [Arize](https://docs.arize.com/arize/) is the machine learning observability platform which can help you to monitor real-time model performance, root cause model failures/performance degradation using explainability & slice analysis and surface drift, data quality, data consistency issues etc.

Reviewer: ```suggestion
[Arize](https://docs.arize.com/arize/) is a machine learning observability platform which can help you to monitor real-time model performance, root cause model failures/performance degradation using explainability & slice analysis and surface drift, data quality, data consistency issues etc.
```

> To integrate with Arize, you can directly add Arize client code into your serve deployment code. ([Example code](https://docs.arize.com/arize/integrations/integrations/anyscale-ray-serve))

Reviewer: ```suggestion
To integrate with Arize, you can directly add Arize client code into your Serve deployment code. ([Example code](https://docs.arize.com/arize/integrations/integrations/anyscale-ray-serve))
```
